fs/expose/dokan/__init__.py

```diff
@@ -262,12 +309,13 @@ def wrapper(self,*args):
 
 MIN_FH = 100
 
+
 class FSOperations(object):
     """Object delegating all DOKAN_OPERATIONS pointers to an FS object."""
 
-    def __init__(self, fs, fsname="Dokan FS", volname="Dokan Volume"):
+    def __init__(self, fs, fsname="NTFS", volname="Dokan Volume"):
```

Comment:
Windows has some internal feature check that only work if the FS is NTFS.
https://github.com/dokan-dev/dokany/blob/0955ed25243f5ba6f0290e1f4207ec5c996a1863/samples/dokan_mirror/mirror.c#L1173-L1175